cdap-ui/server/express.js

```diff
@@ -68,6 +68,13 @@ const getExpressStaticConfig = () => {
   };
 };
 
+const stripAuthHeadersInProxyMode = (cdapConfig, res) => {
+  if (cdapConfig['security.authentication.mode'] === 'PROXY') {
+    delete res.headers.Authorization;
+    delete res.headers[cdapConfig['security.authentication.proxy.user.identity.header']];
+  }
+  return res;
+}
```

Comment:
this seems to be similar to the one in Aggregator. Can it be extracted to a common function?

Reply:
It is slightly different in the sense one is accepting the resource object and the other is accepting response object. Anyhow I have extracted it out to treat it as a generic object.